Death - Legions of Nagash.cat

```diff
@@ -421,6 +421,10 @@
   </entryLinks>
   <sharedSelectionEntries>
     <selectionEntry id="5158-12e9-2204-0818" name="Allegiance" hidden="false" collective="false" import="true" type="upgrade">
+      <constraints>
+        <constraint field="selections" scope="roster" value="1.0" percentValue="false" shared="true" includeChildSelections="true" includeChildForces="false" id="039f-e6db-e483-ba9a" type="min"/>
+        <constraint field="selections" scope="roster" value="1.0" percentValue="false" shared="true" includeChildSelections="true" includeChildForces="false" id="c13a-1b0d-70f2-4678" type="max"/>
```

Comment:
Should the `scope` of this one be `parent`? Seems like the rest follow a `roster/parent` pattern but this is `roster/roster`

Reply:
It won't be functionally any different. That being said, LoN still isn't going to work with the release, mainly because it's using a different Allegiance SSE that I overlooked. Uploading again, but not doing another release.